packages/dashboard/src/components/appbar.tsx

```diff
@@ -451,6 +459,22 @@ export const AppBar = React.memo(({ extraToolbarItems }: AppBarProps): React.Rea
           >
             <Settings />
           </IconButton>
+          <IconButton
+            id="show-help-btn"
+            aria-label="help"
+            color="inherit"
+            onClick={() => window.open(resourceManager?.helpLink, '_blank')}
+          >
+            <Help />
+          </IconButton>
+          <IconButton
+            id="show-warning-btn"
+            aria-label="warning"
+            color="inherit"
+            onClick={() => window.open(resourceManager?.reportIssue, '_blank')}
+          >
+            <Issue />
+          </IconButton>
```

Comment:
ah I noticed how difficult it is to figure out what these buttons do while testing, since it is just an icon. Could you help add tooltips to these buttons and some of the other buttons I listed in the asana ticket?
I'll send over the asana ticket link to you via DM

Reply:
Sure thing!
I added tooltips here f48bbb28e50f7a35f3256e6c06d061fcce24ac72 and I going to add the others in another PR